cerberus-org/cerberus-app
# Typescript for server

# #16 

_**This is dependent on #15 .**_

Oh my god. This was awful to figure out. At least we can now use TypeScript on our backend. The tsc compiler was configured to compile `.ts` in server and output to `dist/server`.

## Testing
1. Run `npm start`
2. Make sure the client displays on `localhost:3000`
3. Make sure the `dist` directory contains `server` and the `.js` files
